ddtrace/tracer.py

```diff
@@ -130,9 +142,10 @@ def __init__(self, url=None, dogstatsd_url=DEFAULT_DOGSTATSD_URL):
         # traces
         self._pid = getpid()
 
+        self.enabled = asbool(environ.get("DD_TRACE_ENABLED", True))
```

Comment:
should we use `asbool(get_env("trace", "enabled", True))` here so `DATADOG_TRACE_ENABLED` will still work?

Reply:
Yes, we should. Was an oversight to not have the DATADOG one included.